tests/warn_conflict/tests/01-make.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import pexpect
+
+BOARD = os.getenv('BOARD', 'stm32f4discovery')
+
+
+def testfunc():
+    cross_gcc = "arm-none-eabi-gcc"
+
+    try:
+        devnull = open(os.devnull)
+        subprocess.Popen([cross_gcc],
+                         stdout=devnull, stderr=devnull).communicate()
+    except OSError as exc:
+        if exc.errno == os.errno.ENOENT:
+            print("ABORTING TEST: {} seems to be missing.\n".format(cross_gcc))
+    else:
+        child = pexpect.spawn(['make'], env=os.environ)
+
+        try:
+            if BOARD == 'stm32f4discovery':
+                child.expect_exact(b'\x1b[1;33mThe following features may conflict:'
+                                   b'\x1b[0m \x1b[1;32mperiph_dac periph_spi\x1b[0m')
```

Comment:
Here and in other lines: I think using `.*` instead of the color codes could be more readable (if it works)

Reply:
(you have to use `expect` instead of `expect_exact` instead though)